assets/js/theme/common/models/forms.js

```diff
@@ -22,6 +22,45 @@ const forms = {
     notEmpty(value) {
         return value.length > 0;
     },
+
+    /**
+     * validates a field like product quantity
+     * @param value
+     * @returns {boolean}
+     *
+     */
+    areNumbersOnly(value) {
+        const re = /^\d+$/;
+        return re.test(value);
+    },
```

Comment:
I believe it should better name for example as `checkAreNumbersOnly`

Reply:
it's been made with analogy to `notEmpty` method above. I assume we can change to `numbersOnly` if you prefer more.

Reply:
`numbersOnly` sounds good